File: front-end/src/components/event/Event.js
class Event {
  constructor(
    studioShortName,
    studioLongName,
    shortName,
    longName,
    year,
    duration,
    licensePrice,
    type
  ) {
    this.studioShortName = studioShortName;
    this.studioLongName = studioLongName;
    this.shortName = shortName;
    this.longName = longName;
    this.year = year;
    this.duration = duration;
    this.licensePrice = licensePrice;
    this.type = type;
  }
}

export default Event;

File: back-end/src/main/java/com/model/event/Event.java
package com.model.event;

import java.util.Date;


public abstract class Event {
    public String shortName;
    public String longName;
    public Date dateCreated;
    public Date year;
    public double duration;
    public int number;

    public Event(String shortName, String longName, Date dateCreated, Date year, double duration,int number) {
        this.shortName = shortName;
        this.longName = longName;
        this.dateCreated = dateCreated;
        this.year = year;
        this.duration = duration;
        this.number = number;
    }




}
